pptxProject.py

```diff
@@ -0,0 +1,127 @@
+import os
```

Comment:
you didn't type you're function params, and type the return value

Reply:
I'll fix it